src/main/java/pl/winiarczyk/lesson4/homework/RightTriangle/Triangle.java

```diff
@@ -0,0 +1,18 @@
+package pl.winiarczyk.lesson4.homework.RightTriangle;
+
+public class Triangle {
+
+    private static int sideA;
+    private static int sideB;
+    private static int sideC;
```

Comment:
czemu to jest static? To mogą być zwykle pola - ustawiasz je w konstruktorze. Ten static jest tutaj zbędny.